[FEDEXSHIP-35] Update Docs (#31)
vtex-apps/fedex-shipping: README.md

> 📢 Use this project, [contribute](https://github.com/vtex-apps/packing-optimization) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).

Reviewer: ```suggestion
📢 Use this project, [contribute](https://github.com/vtex-apps/fedex-shipping) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).

```

# FedEx Shipping
<!-- DOCS-IGNORE:start -->
<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
[![All Contributors](https://img.shields.io/badge/all_contributors-0-orange.svg?style=flat-square)](#contributors-)
<!-- ALL-CONTRIBUTORS-BADGE:END -->
<!-- DOCS-IGNORE:end -->

> FedEx Dynamic Shipping Rates

Reviewer: ```suggestion
```

> FedEx App works in tandem with shipping-rates-provider to fetch for dynamic rates. This app integrates the FedEx API.

Reviewer: ```suggestion
The **FedEx Shipping** app is an integration with the VTEX [Dynamic Rates Hub](https://github.com/vtex/shipping-rates-provider) to perform real-time shipping cost calculations for the user in checkout. This app uses the FedEx API to calculate shipping costs.

```

## Features 🚚
- Fetch dynamic rates from FedEx

## Installation ⏬
- Set Up a FedEx Developer Account

> - Install the following two apps: `vtex.shipping-rates-provider` and `vtex install vtexus.fedex-shipping`

Reviewer: ```suggestion
- Install the following two apps: `vtex.shipping-rates-provider` (the Dynamic Rates Hub app) and `vtexus.fedex-shipping` (this app) by running `vtex install vtex.shipping-rates-provider` and `vtex install vtexus.fedex-shipping` in your terminal, using the [VTEX IO CLI](https://developers.vtex.com/vtex-developer-docs/docs/vtex-io-documentation-vtex-io-cli-installation-and-command-reference).
```

> - You can install the **Shipping Rates Provider and FedEx App** app by running `vtex.shipping-rates-provider` and `vtex install vtexus.fedex-shipping`, respectively, in your terminal, using the [VTEX IO CLI](https://developers.vtex.com/vtex-developer-docs/docs/vtex-io-documentation-vtex-io-cli-installation-and-command-reference).

Reviewer: ```suggestion
```

> ## Configurations ⚙️

Reviewer: I think it's a good idea to add some instructions on how to find the admin panel, similar to what you have in the Packing Optimization readme.

- You can check dock settings here: `https://{workspace}--{account}.myvtex.com/admin/shipping-strategy/loading-docks/`
- Items coming from the same dock will be treated as if they are shipping together, unless an above setting prevents them from doing so.

### Optimize Shipping 📦
- There are 3 choices for optimizing shipping. Some may reduce shipping costs.
    - None: Treats every item as independent. Highest possible cost.
    - Pack All Into Largest Box: Packs all items into the largest item's box. Lowest possible cost since all items are combined into the dimensions of the largest item. However, this may be unreasonable as space in the box may already be preoccupied.
    - ⚠️***`BETA`*** Smart Packing: Given a list of boxes, we can dynamically pack items into the box, with respect to the space that is already consumed by other items in the box. This feature requires the [Packing Optimization App](https://github.com/vtex-apps/packing-optimization). Please check the [Packing Optimization App](https://github.com/vtex-apps/packing-optimization) for more information on how to use. Most optimal shipping cost.

### Things To Note ⚠️

> > Please `Save` in the current settings tab before navigating to another settings tab.

Reviewer: ```suggestion
> Remember to `Save` in the current settings tab before navigating to another settings tab. Otherwise, your changes will be lost.

```

> The FedEx dynamic rates can only be as good as the data inputted. Please correct the item's dimensions and weights and select the proper unit of measurement before using this app.

> > Item length, width, and height are rounded up.

Reviewer: ```suggestion
> Item length, width, and height are rounded up to the nearest whole number.

```
